docs/user/build-customization.rst

```diff
@@ -326,12 +326,13 @@ Take a look at the following example:
      jobs:
        post_create_environment:
          # Install poetry
-         # https://python-poetry.org/docs/#osx--linux--bashonwindows-install-instructions
-         - curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+         - pip install poetry
          # Tell poetry to not use a virtual environment
-         - $HOME/.poetry/bin/poetry config virtualenvs.create false
-         # Install project's dependencies
-         - $HOME/.poetry/bin/poetry install
+         - poetry config virtualenvs.create false
+       post_install:
```

Comment:
We should remove this `post_install` here to simplify the config a little since it's not required for the most common use case. I tried this in `test-builds` project and it worked properly https://readthedocs.org/projects/test-builds/builds/18704586/

Reply:
I thought it was best to cover more edge cases, in the hope that it would avoid potential issues for old projects and therefore support requests/GitHub issues.

That being said, I don't mind either way & I'll remove this section 👍🏻